docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,114 @@
+# Release 381 (11 May 2022)
+
+## General
+
+* Increase the default number of stages allowed for a query from 100 to 150,
```

Comment:
@arhimondr here is the reworded entry .. please confirm

Reply:
LGTM